sequence.go

```diff
@@ -92,3 +92,20 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+func anonymized_record(sequence Sequence, sequence_id string) Sequence { 
```

Comment:
Rename AnonymizeRecord.

Reply:
Also add docstring comment above:

``//AnonymizeSequence takes a sequence struct and deletes all meta data and returns a Sequence struct with just the sequence string.``